package.json

```diff
@@ -5,7 +5,7 @@
     "Tom Tornese",
     "Jason Aslakson"
   ],
-  "version": "1.0.0",
+  "version": "1.3.0",
```

Comment:
I think we can leave this at 1.0.0 until we actually start using it on the site.